controllers/socialLoginControllers.js

```diff
@@ -57,6 +57,26 @@ class SocialMediaController {
     const status = created ? RESOURCE_CREATED_CODE : OK_CODE;
     res.status(status).send(userDetails);
   }
+
+  /**
+   * @static
+   * @param {object} req - HTTP request object
+   * @param {object} res - HTTP response object
+   * @returns {object} - returns generated token to the redirected url
+   */
+  static async getUserTokenTwitter(req, res) {
+    if (!req.user) {
+      return res.status(BAD_REQUEST_CODE).send(INVALID_USER);
+    }
+    const { id, created } = req.user.user;
+    const token = await generateToken(TOKEN_TIMESPAN, id);
+    Object.assign({}, req.user, {
+      msg: created ? WELCOME_NEW_USER : WELCOME_EXISTING_USER,
+      token
+    });
+    const status = created ? RESOURCE_CREATED_CODE : OK_CODE;
+    res.status(status).redirect(`${process.env.FRONT_URL_TWITTER}=${token}`);
```

Comment:
`FRONT_URL_TWITTER` Can we have the frontend URL as a parament that is sent along with the request from the front-end? Having it the URL as a parameter will ensure that this feature/endpoint is not coupled with our frontend application.

Reply:
I'll look into it